hack/generate-install-script.sh

```diff
@@ -35,4 +35,8 @@ write_block "sql/schema.sql"       sql/schema.sql
 echo "### TEMPLATES" >> $dst
 write_block "templates/hello-world-now.yaml" ./examples/templates/hello-world-now.yaml
 
+version=`git describe --tags $(git rev-list --tags --max-count=1)`
+sed -i.bak "s/DOCKER_TAG/$version/" ./$dst
```

Comment:
No need to create a backup file if it is deleted just after ;)
```suggestion
sed -i "s/DOCKER_TAG/$version/" ./$dst
```

Reply:
incredible as it may sound, it's there for compatibility with mac os :|